app/uk/gov/hmrc/customs/declaration/controllers/actionbuilders/upscan/fileUploadPayloadValidationActions.scala

```diff
@@ -83,7 +83,7 @@ class FileUploadPayloadValidationComposedAction @Inject()(val fileUploadPayloadV
             val declarationId = DeclarationId((xml \ declarationIdLabel).text)
             val fileGroupSize = FileGroupSize((xml \ fileGroupSizeLabel).text.trim.toInt)
 
-            val files: Seq[FileUploadFile] = (xml \ filesLabel \ "_").theSeq.collect {
+            val files: scala.collection.Seq[FileUploadFile] = (xml \ filesLabel \ "_").theSeq.collect {
```

Comment:
Can we leave the type annotation as it is and not call `.theSeq`? You'll see that `scala.xml.NodeSeq` already extends `scala.collection.immutable.Seq`.